### PR TITLE
Minor changes in email notifications and Greek Translation

### PR DESCRIPTION
**Pull request by saggel**
I submit some minor editing on the english confirmation instructions -
mostly as a test before I submit my Greek translation!

### CODE REVIEW

File: app/views/user_mailer/password_change.en.html.erb [hunk 1]
```diff
@@ -1,3 +1,3 @@
 <p>Hello <%= @resource.email %>!</p>
 
-<p>We're contacting you to notify you that your password on Mastodon has been changed.</p>
+<p>We're contacting you to notify you that your password on <%= @instance %> has been changed.</p>
```
Comment (mjankowski) [inline app/views/user_mailer/password_change.en.html.erb]: Can you put this change in a separate PR?

File: config/application.rb [hunk 2]
```diff
@@ -30,6 +30,7 @@ class Application < Rails::Application
       :ar,
       :bg,
       :de,
+	  :el,
```
Comment (mjankowski) [inline config/application.rb]: alignment?

File: config/locales/doorkeeper.el.yml [hunk 3]
```diff
@@ -0,0 +1,113 @@
+---
+el:
```
Comment (mjankowski) [inline config/locales/doorkeeper.el.yml]: did anything in this file change from en?

File: config/locales/el.yml [hunk 4]
```diff
@@ -0,0 +1,341 @@
+---
+el:
```
Comment (mjankowski) [inline config/locales/el.yml]: did anything in this file change from en?

**Review by mjankowski — CHANGES_REQUESTED** [review]

File: config/locales/el.yml (hunk 4 above)
Comment (saggel) [inline config/locales/el.yml]: The Greek code is el - did I do something wrong?

File: config/locales/doorkeeper.el.yml (hunk 3 above)
Comment (saggel) [inline config/locales/doorkeeper.el.yml]: The Greek code is el - did I do something wrong?

File: config/application.rb (hunk 2 above)
Comment (saggel) [inline config/application.rb]: I can fix this on the next pull where I'll complete the translation!

File: config/locales/el.yml (hunk 4 above)
Comment (mjankowski) [inline config/locales/el.yml]: It looks like nothing change in the file...

The `en` changed to `el`, which is correct ... but it looks like nothing was translated?

File: app/views/user_mailer/password_change.en.html.erb (hunk 1 above)
Comment (saggel) [inline app/views/user_mailer/password_change.en.html.erb]: If you tell me how :-)

**Comment by saggel** [comment]
If you think I should just do the pull again right from the beginning please do let me know! It was my first attempt!

File: config/locales/el.yml (hunk 4 above)
Comment (saggel) [inline config/locales/el.yml]: Yes that's true - not done yet, but I thought it needs to be on the folder for the locale to work - the translation is done on 85% and I wanted to try it on an instance before completing it!

**Comment by mjankowski** [comment]
Congrats on your first PR and thanks for the help! ... 

- The formatting changes to the mailers look good
- I'm not sure the change which adds `@instance` to the password change mailer will work, because I don't think that variable is in place in that mailer...
- I think the translations will just fall back to the `en` versions if some are missing

I think my preference would be to do one PR for the mailer changes, and another for all of the added `el` locale additions (including el mailers).

If you're working purely through the GH website and not a local copy, I can help create the mailer PR separately, and then we can update this one to only include the el changes.

**Comment by saggel** [comment]
Awesome - let me know what I need to do - I use the github desktop app - but happy to do whatever is easier or faster for you!

**Comment by yookoala** [comment]
Your works is based on an older version of Mastodon. Can you please rebase your work on the latest master branch? Or at least merge with the latest Mastodon's master?

**Comment by Gargron** [comment]
Closing for inactivity

**Pull request closed without merging**